### ISSUE TITLE
No return value when we perform get operation on a stratum device 

### ISSUE THREAD

**Issue by adibrastegarnia** [onosproject/onos-config#916]
**Describe the bug**
Wrong return value when we perform a get operation on a leaf value of a stratum device:

**To Reproduce**
1. onit create cluster onit-1
2. onit add network network-1
3. onit onos-cli
4. Run the following command from inside onos-cli:
```
   gnmi_cli -address onos-config:5150 -get \
    -proto "path: <target: 'network-1-s0', elem: <name:'interfaces'> elem:<name:'interface' key: <key: 'name' value: 's1-eth1'>> elem:<name:'config'> elem:<name:'enabled'>>" \
    -encodingType "PROTO" \
```

Output is the following,
```
notification: <
  timestamp: 1574544332
  update: <
    path: <
      elem: <
        name: "interfaces"
      >
      elem: <
        name: "interface"
        key: <
          key: "name"
          value: "s1-eth1"
        >
      >
      elem: <
        name: "config"
      >
      elem: <
        name: "enabled"
      >
      target: "network-1-s0"
    >
  >
>
```

I would expect to see if that field is true or false. If you run the above command on the device directly the output will be like the following:

```
   gnmi_cli -address network-1-s0 -get \
    -proto "path: <elem: <name:'interfaces'> elem:<name:'interface' key: <key: 'name' value: 's1-eth1'>> elem:<name:'config'> elem:<name:'enabled'>>" \
    -encodingType "PROTO" \
   -tlsDisabled \
```

```
notification: <
  timestamp: 1574544430810409497
  update: <
    path: <
      elem: <
        name: "interfaces"
      >
      elem: <
        name: "interface"
        key: <
          key: "name"
          value: "s1-eth1"
        >
      >
      elem: <
        name: "config"
      >
      elem: <
        name: "enabled"
      >
    >
    val: <
      bool_val: true
    >
  >
>
```

**Comment by adibrastegarnia** [comment]
I see the same pattern with onos-config? Am I missing something? Should I enable anything. It looks like plugins already loaded successfully.

**Comment by Andrea-Campanella** [comment]
@adibrastegarnia we currently do not load the initial configuration present in a device, that is done because we assume ONOS is the master of the configuration of the network. If `onos-config` had a configuration for that device it will override the existing one. 
We could work on a functionality that performs an initial get on the device, does a diff agains the existing config, saves the ones that `onos-config` does not have while applies the one `onos-config` has. @SeanCondon thoughts ?

**Comment by SeanCondon** [comment]
Yeah. @adibrastegarnia I think the title is misleading here - what is being returned is **no** value. And as Andrea says this is not a surprise - the design of the system is that `onos-config` is the master of the configuration and you have not put anything in to `onos-config` for that device. The device has values but they are wrong - because they are not what `onos-config` says they should be i.e. blank!

What @Andrea-Campanella mentions is some optional new behaviour that we could add as a new feature - i.e. a init-synchronization - do we want this?

**Comment by adibrastegarnia** [comment]
You are right about master of configuration but I think independent of that, some sort of synchronization is also needed between what we have in onos-config and the device even initially (suppose there is no onos-ztp or any other systems to provision the configuration). In addition, if my understanding is correct (correct me if I am wrong), onos-config should relay on some of the values from the device itself. For example, packet counters, number of interfaces (i.e. state variables). If my understanding is correct, onos-config cannot touch them because the network determines them depending on the topology and traffic

**Comment by Andrea-Campanella** [comment]
Closing, will be addressed with to #917
